src/server/utils/create-ssr-html.tsx

```diff
@@ -73,7 +73,7 @@ export async function createSsrHtml(
     <!-- Required meta tags -->
     <meta name="Description" content="Lemmy">
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=no">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=yes">
```

Comment:
We can also just omit `user-scalable` entirely since it defaults to `yes`.

Reply:
Omitted.